build-system/eslint-rules/no-forbidden-terms.js

```diff
@@ -33,7 +35,9 @@ module.exports = function (context) {
         const {match, message, loc} = report;
```

Comment:
This will need a `require` at the top:

```js
const {cyan} = require('kleur/colors');
```

```suggestion
        const {match, message, loc} = report;
        const formattedMatch = cyan(`"${match}"`);
```